docs/text/rolling.rst

```diff
@@ -0,0 +1,160 @@
+.. _rolling-label:
+
+How to handle rolling time series
+=================================
+
+In many application with time series on real-world problems, the "time" column
+(we will call it time in the following, although it can be anything)
+gives a certain sequential order to the data. We can exploit this sequence to generate
+more input data out of single time series, by *rolling* over the data.
+
+Imagine the following situation: you have the data of EEG measurements, that
```

Comment:
this example confused me a little bit. 

E.g. what do you mean by "if you have stopped the measurement"?